grammar.js

```diff
@@ -146,7 +147,8 @@ module.exports = grammar({
       )
     ),
 
-    enum_case_definitions: $ => seq(
+    enum_case_definition: $ => seq(
```

Comment:
Why was this renamed here? Since it's a `seq(...)` wouldn't it make sense to keep it plural?

Reply:
Hm
I read it as a "definition of a single enum case", hence i'd prefer it singular 🤔 
One might think that this rule handles definitions of all enum cases
EBNF calls it just EnumCase

Reply:
These names leak out to be tree-sitter queries in various editors, so unless there's a good reason we should avoid renaming them, and even if we do try to make an alias to keep backward compatibility.

Reply:
Reverted the rename